app/javascript/controllers/clipboard2_controller.ts

```diff
@@ -82,9 +84,16 @@ export class Clipboard2Controller extends Controller {
     this.#timer = setTimeout(() => {
       this.#copiedSpan.remove();
     }, SUCCESS_MESSAGE_TIMEOUT);
+
+    wrapper.addEventListener('mouseleave', () => {
+      this.#copiedSpan.remove();
+    });
   }
 
   private insertSpan(wrapper: HTMLElement, span: HTMLElement): void {
+    // Remove span from DOM if it's already attached from a previous insertion
+    span.remove();
```

Comment:
alors, j'avais compris qu'un element ne pouvais pas être a 2 endroits au mm endroit https://developer.mozilla.org/fr/docs/Web/API/Node/appendChild . Du coup, je suis pas sur que le remove soit nécessaire. pareil pour la ligne 89.

Reply:
Je ne comprends pas très bien : j avais l impression que le `span` était automatiquement déplacé par le dom : `si le nœud a déjà un parent, le nœud est d'abord retiré, puis ajouté à la nouvelle position`

Du coup ta ligne serait redondante avec les `appendChild` un peu plus loin.

Pour le coup du double click (well spotted), j ai l'impression qu'on peut corriger, en mettant un `this.#copiedSpan.remove();` ligne 65. wdyt ?